api/controllers/UserController.js

```diff
@@ -248,10 +254,17 @@ module.exports = {
       }
     )
     var total
-
     Search.forUsers(options).fetchAll({withRelated: ['skills', 'organizations', 'memberships']})
     .tap(users => total = (users.length > 0 ? users.first().get('total') : 0))
     .then(users => users.map(u => UserPresenter.presentForList(u, res.locals.community.id)))
+    .then(list =>
```

Comment:
@levity - would you take a look at how I'm getting the createdTags and the offerCount for members page here. I couldn't decide between this and putting it in to SearchForUsers / UserPresenter.
Let me know if you have a preference